searcher.proto

```diff
@@ -93,6 +93,8 @@ message GetTipAccountsResponse {
 
 message SubscribeBundleResultsRequest {}
```

Comment:
possible to set as deprecated? https://stackoverflow.com/a/52787662/3408577

Reply:
Yeah, set as deprecated. Unfortunately this does not generate any compiler warning. Only way to generate compiler warning is by setting [deprecated = true] for individual fields. To note: setting [deprecated = true]does not work on "oneof" fields.